docs/setup/kubeedge_configure.md

```diff
@@ -0,0 +1,369 @@
+      enable_uds: true # enable unix domain socket protocol
+      uds_address: unix:///var/lib/kubeedge/kubeedge.sock # unix domain socket address
+      address: 0.0.0.0
+      ca: /etc/kubeedge/ca/rootCA.crt
+      cert: /etc/kubeedge/certs/edge.crt
+      key: /etc/kubeedge/certs/edge.key
+      keepalive-interval: 30
+      write-timeout: 30
+      node-limit: 10
+    devicecontroller:
+      kube:
+        master:        # kube-apiserver address (such as:http://localhost:8080)
+        namespace: ""
+        content_type: "application/vnd.kubernetes.protobuf"
+        qps: 5
+        burst: 10
+        kubeconfig: "/root/.kube/config" #Enter absolute path to kubeconfig file to enable https connection to k8s apiserver,if master and kubeconfig are both set, master will override any value in kubeconfig.
+```
+
+cloudcore default supports https connection to Kubernetes apiserver, so you need to check whether the path for `controller.kube.kubeconfig` and `devicecontroller.kube.kubeconfig` exist, but if `master` and `kubeconfig` are both set, `master` will override any value in kubeconfig.
+
+Check whether the cert files for `cloudhub.ca`, `cloudhub.cert`,`cloudhub.key` exist.
+
+#### Modification in controller.yaml
+
+In the controller.yaml, modify the below settings.
+
+1. `controller.kube.master` : This would be kube-apiserver address. It might be 
+
+    ```
+    https://<your_hostname>:6443
+    ```
+    or
+    ```
+    http://<your_hostname>:8080
+    ```
+    based on your kubernetes configuration. `your_hostname` should be replaced with the IP Address of your hostname.
+
+2. `controller.kube.kubeconfig` and `devicecontroller.kube.kubeconfig` : This would be the path to your kubeconfig file. It might be either
+
+    ```
+    /root/.kube/config
+    ```
+    or
+    ```
+    /home/<your_username>/.kube/config
+    ```
+    depending on where you have setup your kubernetes by performing the below step:
+
+    ```
+    To start using your cluster, you need to run the following as a regular user:
+
+    mkdir -p $HOME/.kube
+    sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
+    sudo chown $(id -u):$(id -g) $HOME/.kube/config
+    ```
+
+### Adding the edge nodes (KubeEdge Worker Node) on the Cloud side (KubeEdge Master Node)
+
+We have provided a sample node.json to add a node in kubernetes. Please make sure edge-node is added in kubernetes. Run below steps to add edge-node.
+
++ Copy the `$GOPATH/src/github.com/kubeedge/kubeedge/build/node.json` file and change `metadata.name` to the name of the edge node
+
+    ```shell
+        mkdir -p ~/cmd/yaml
+        cp $GOPATH/src/github.com/kubeedge/kubeedge/build/node.json ~/cmd/yaml
+    ```
+
+    Node.json
+    ```script
+    {
+      "kind": "Node",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "edge-node",
+        "labels": {
+          "name": "edge-node",
+          "node-role.kubernetes.io/edge": ""
+        }
+      }
+    }
+    ```
```

Comment:
perhaps also mention the following:

If you want to add an edge node to an existing kubernetes cluster you should a taint to this node. An example is given in this Node.yaml file:

```
kind: Node
apiVersion: v1
metadata:
  name: edge-node
  labels:
    name: edge-node
    node-role.kubernetes.io/edge: ""
spec
  taints
    effect: noSchedule
    key: node.kubeedge.io
    value: edge
```